src/components/Dropdown.tsx

```diff
@@ -12,6 +13,8 @@ type DropdownProps = {
   setDropdown: (dropdown?: string) => void;
   setState: (state: State) => void;
   state: State;
+  setSearchParams: SetURLSearchParams;
+  searchParams: URLSearchParams;
 };
```

Comment:
i'd take out `setState`, `setSearchParams`, and `searchParams` out of here

Reply:
👍